### PR TITLE
Holy hyperlinks

### PR DESCRIPTION
**Pull request by NightNutSky**

### CODE REVIEW

File: src/bdscript/day.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # $day
 Returns the current day of the month.
-> 🧙‍♂️ You can use [$time](https://nilpointer-software.github.io/bdfd-wiki/bdscript/time.html) to change the timezone.
+> 🧙‍♂️ You can use [$time](./bdscript/time.md) to change the timezone.
```
Comment (noituri) [inline src/bdscript/day.md:3]: Invalid hyperlink. Should be `./time.md`

File: src/bdscript/hour.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # $hour
 Returns the current minute.
-> 🧙‍♂️ You can use [$time](https://nilpointer-software.github.io/bdfd-wiki/bdscript/time.html) to change the timezone.
+> 🧙‍♂️ You can use [$time](./bdscript/time.md) to change the timezone.
```
Comment (noituri) [inline src/bdscript/hour.md:3]: Invalid hyperlink

File: src/bdscript/minute.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # $minute
 Returns the current minute of this hour.
-> 🧙‍♂️ You can use [$time](https://nilpointer-software.github.io/bdfd-wiki/bdscript/time.html) to change the timezone.
+> 🧙‍♂️ You can use [$time](./bdscript/time.md) to change the timezone.
```
Comment (noituri) [inline src/bdscript/minute.md:3]: Invalid hyperlink

File: src/bdscript/month.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # $month
 Returns the current month of this year.
-> 🧙‍♂️ You can use [$time](https://nilpointer-software.github.io/bdfd-wiki/bdscript/time.html) to change the timezone.
+> 🧙‍♂️ You can use [$time](./bdscript/time.md) to change the timezone.
```
Comment (noituri) [inline src/bdscript/month.md:3]: Invalid hyperlink

File: src/bdscript/second.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # $second
 Returns the current second of this minute.
-> 🧙‍♂️ You can use [$time](https://nilpointer-software.github.io/bdfd-wiki/bdscript/time.html) to change the timezone.
+> 🧙‍♂️ You can use [$time](./bdscript/time.md) to change the timezone.
```
Comment (noituri) [inline src/bdscript/second.md:3]: Invalid hyperlink

File: src/bdscript/year.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # $year
 Returns the current year.
-> 🧙‍♂️ You can use [$time](https://nilpointer-software.github.io/bdfd-wiki/bdscript/time.html) to change the timezone.
+> 🧙‍♂️ You can use [$time](./bdscript/time.md) to change the timezone.
```
Comment (noituri) [inline src/bdscript/year.md:3]: Invalid hyperlink

**Review by noituri — CHANGES_REQUESTED** [review]

**Comment by NightNutSky** [comment]
I fell into a coffin 💀

**Review by noituri — APPROVED** [review]

**Pull request merged**